pkg/logs/internal/launchers/file/launcher.go

```diff
@@ -60,10 +61,20 @@ type Launcher struct {
 }
 
 // NewLauncher returns a new launcher.
-func NewLauncher(tailingLimit int, tailerSleepDuration time.Duration, validatePodContainerID bool, scanPeriod time.Duration) *Launcher {
+func NewLauncher(tailingLimit int, tailerSleepDuration time.Duration, validatePodContainerID bool, scanPeriod time.Duration, wildcardMode string) *Launcher {
+	var prov *fileprovider.FileProvider
+	if wildcardMode == "prioritizeNewest" {
+		prov = fileprovider.NewFileProvider(tailingLimit, fileprovider.WildcardMtime, fileprovider.GlobalSelection)
+	} else if wildcardMode == "simpleAlpha" {
+		prov = fileprovider.NewFileProvider(tailingLimit, fileprovider.WildcardReverseLexicographical, fileprovider.GreedySelection)
+	} else {
+		log.Warnf("Unknown wildcard mode specified: %q, defaulting to 'simpleAlpha' strategy.\n", wildcardMode)
```

Comment:
NIT: Do we need the `\n`? Most of our other `Warnf` messages don't append a newline (but a few do).

Reply:
Every log goes through the scrubber which will by default add a trailing newline, so this doesn't appear necessary. I'll remove it for consistency